example.py

```diff
@@ -21,8 +21,6 @@
 torch_normalizer = torchstain.normalizers.MacenkoNormalizer(backend='torch')
```

Comment:
This example does not need to be adjusted. It is meant to showcase all backends (tf included). Hence, revert the changes in this file and make a new commit.

Reply:
Fixed